client.py

```diff
@@ -0,0 +1,74 @@
+import tkinter as tk
+import requests
```

Comment:
Add `requests` to `requirements.txt` because it is a third-party package. I don't have the packages that you installed, and `requirements.txt` is a file that gives the project list of all the packages that Python needs to install in order to run it.